extensions/windows/cfapi/Common/utilities.cpp

```diff
@@ -17,12 +17,12 @@
  */
 
 #include "utilities.h"
+#include "../../../../src/libcommon/utility/utility_base.h"
```

Comment:
Is the part `../../../../src/`really necessary?

Reply:
Try to link the extension with libcommon library.

Reply:
Would it be OK to keep things like this and create a common library that can be shared with extensions in a second step? I have written the following Jira ticket : https://infomaniak.atlassian.net/browse/KDESKTOP-1435 regarding the creation of a new utility library.

Reply:
OK